textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/controller/executors/SakaiTmbCommandExecutor.java

```diff
@@ -29,12 +29,14 @@
     public void execute(FsService fsService, HttpServletRequest request, HttpServletResponse response, ServletContext servletContext) throws Exception {
         String target = request.getParameter("target");
         FsItemEx fsi = super.findItem(fsService, target);
-        int width = fsService.getServiceConfig().getTmbWidth();
-        try (InputStream is = fsi.openInputStream()) {
-            ByteArrayOutputStream baos = resize(is, width);
-            response.setHeader(HttpHeaders.LAST_MODIFIED, DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now().withZoneSameInstant(ZoneId.of("GMT"))));
-            response.setHeader(HttpHeaders.EXPIRES, DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now().withZoneSameInstant(ZoneId.of("GMT")).plus(2, ChronoUnit.YEARS)));
-            response.getOutputStream().write(baos.toByteArray());
+        if (fsi.getSize() < 20971520) { // image must be less than 20MB
```

Comment:
FY I didn't opt to creating a config option for this as it is simply a reasonable upper limit and injecting the services into these classes is not trivial due to the way they are instantiated.

Reply:
Okay, can we make this a constant at least?